aya/src/maps/perf/perf_event_array.rs

```diff
@@ -207,4 +212,27 @@ impl<T: BorrowMut<MapData>> PerfEventArray<T> {
             _map: self.map.clone(),
         })
     }
+
+    /// Inserts a perf_event file descriptor at the given index.
+    ///
+    /// ## Errors
+    ///
+    /// Returns [`MapError::OutOfBounds`] if `index` is out of bounds, [`MapError::SyscallError`]
+    /// if `bpf_map_update_elem` fails.
+    pub fn set<FD: AsFd>(&mut self, index: u32, value: &FD) -> Result<(), MapError> {
```

Comment:
What happens if I set an index that has already been `open()`-ed?

Reply:
Well, I have not tested this case. If the fd has already been opened, it could stay open? The doc does not give any clear hint about this case.